02-Git/Solution/game.c

```diff
@@ -8,6 +8,8 @@ int main(){
     srand(time(NULL));
     const char roll[] = {'r','p','s'};
     char choose_pc[10];
+    
+    unsigned int stat[2];
```

Comment:
* applies to entire commit, not this lines

1. no binary files should be added
2. no description why this statistics is needed